lib/grid_creator.rb

```diff
@@ -0,0 +1,23 @@
+module GridCreator
+
+  BOX_OF_INDEX = [
+      0,0,0,1,1,1,2,2,2,0,0,0,1,1,1,2,2,2,0,0,0,1,1,1,2,2,2,
+      3,3,3,4,4,4,5,5,5,3,3,3,4,4,4,5,5,5,3,3,3,4,4,4,5,5,5,
+      6,6,6,7,7,7,8,8,8,6,6,6,7,7,7,8,8,8,6,6,6,7,7,7,8,8,8
+    ]
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.